packages/core/src/Stores/Modules/Cashier/cashier-store.js

```diff
@@ -1010,4 +1010,12 @@ export default class CashierStore extends BaseStore {
 
         return Promise.resolve();
     }
+
+    onThemeChangeListener() {
+        const is_deposit = this.active_container === 'deposit';
```

Comment:
we would need it for withdrawal too actually, but then it would invalidate your token and you would need the verification email all over again.

Reply:
@negarn I think changing theme while depositing/withdrawing is such a rare event that we could consider skipping this all together to prevent the extra complexity as it still works.